features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/RumConfiguration.kt

```diff
@@ -283,6 +284,20 @@ data class RumConfiguration internal constructor(
             return this
         }
 
+        /**
+         * Sets the configuration for the slow frames listener, which is used to compute the slow frames array,
+         * slow frames ratio, and freeze ratio.
```

Comment:
Should we explain the things a bit more in the details? As a customer I don't understand what each of this means: `slow frames array, slow frames ratio, and freeze ratio.`

Reply:
Good point, but I guess we need separate documentation  page  for it. I plan to add link to it later.